tests/spec-test.toit

```diff
@@ -9,7 +9,10 @@ import host.file
 import mustache
 
 main args:
-  specs-path := args.first
+  specs-path/string := args.first
+  if specs-path.ends-with "toit" or specs-path.ends-with "toit.exe":
```

Comment:
Maybe add a comment that explains why this is likely to be an indication that we're called from the toit repository? It isn't super obvious to me.